torch_patches/.torch_pin

```diff
@@ -0,0 +1 @@
+680a89e2ced8f45b87b7fd0509c07cf6b768be14
```

Comment:
why do we need a pin? I thought Brian's change already merged to main.

Reply:
It was reverted.